packages/@glimmer/compiler/lib/builder/builder-interface.ts

```diff
@@ -322,7 +322,7 @@ function normalizeVerboseStatement(statement: VerboseStatement): NormalizedState
 }
 
 function extractBlockHead(name: string): NormalizedHead {
-  const result = /^(#|!)(.*)$/u.exec(name);
+  const result = /^(#|!)(.*)$/v.exec(name);
```

Comment:
why should any of these be using unicode?

![image](https://github.com/user-attachments/assets/08b0b02c-f209-4c3e-80e8-e6ea02f6d4be)


This lint made everything slower

Reply:
bench here: https://github.com/NullVoxPopuli/node-benchmarks/blob/main/tests/glimmer-vm-isElement.js

Reply:
the `/u` variant is pretty close to native (sometimes)
![image](https://github.com/user-attachments/assets/972ae17d-5774-4657-bd21-21b277e32181)

Reply:
finally found a tool that expresses the information in a way I like:
![image](https://github.com/user-attachments/assets/1985244b-5baa-489c-a7f3-4982acd88c16)

Reply:
I think `/u` is a good general practice (because non-unicode regexes have surprising behavior and `/u` is a pretty free-and-clear upgrade without huge compat headaches). See https://eslint.org/docs/latest/rules/require-unicode-regexp.

`/v` is also touted as a free-and-clear upgrade to `/u`, but it's quite new, seems to have perf implications, and is a huge change to the character class syntax. With time, I think it will be the right default, but I agree that that time is not now, and plan to revert this change and update the lint.